src/templates/LectureView/index.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { graphql } from 'gatsby'
+import { graphql, Link } from 'gatsby'
 
 import Layout from '../../components/Layout'
 import LecturesList from '../../components/LecturesList'
 import Seo from '../../components/Seo'
 
 import { StyledPlayerWrap, StyledPlayer } from './styles'
+import { AiFillForward, AiFillBackward } from 'react-icons/ai'
 
 const LectureView = ({ data }) => {
   const { strapiLecture, strapiCourse } = data
-  const { url } = strapiLecture
+  const { url, title, position } = strapiLecture
+  const { title: courseTitle } = strapiCourse
+
+  const findLectureByPosition = index => {
+    return strapiCourse.lectures.find(
+      lecture => lecture.position === position + index
+    ).id
+  }
```

Comment:
can you make this defensive, by checking that the result of find is not null, before trying to do `.id`

Reply:
done